typescript/looker/sdk/3.1/streams.ts

```diff
@@ -2152,9 +2148,7 @@ export class Looker31SDKStream extends APIMethods {
     return this.authStream<IRenderTask>(
       callback,
       'POST',
-      encodeURI(
-        `/render_tasks/lookml_dashboards/${request.dashboard_id}/${request.result_format}`,
-      ),
+      `/render_tasks/lookml_dashboards/${request.dashboard_id}/${request.result_format}`,
```

Comment:
if result_format is a string should it be getting url encoded?

Reply:
fixed